pagerenderer/vue/ui.apps/src/main/content/jcr_root/apps/pagerender/vue/structure/page/og_tag_dialog.json

```diff
@@ -0,0 +1,68 @@
+{
+  "fields": [
+    {
+      "type": "material-checkbox",
+      "label": "Use site-title as og:title?",
+      "default": true,
+      "model": "useTitle"
+    },
+    {
+      "type": "input",
+      "inputType": "text",
+      "model": "ogTitle",
+      "label": "og:title",
+      "placeholder": "og:title",
+      "visible": "model.useTitle != true"
+    },
+    {
+      "type": "material-checkbox",
+      "label": "Use site-description for og:description",
+      "default": true,
+      "model": "useDescription"
+    },
+    {
+      "type": "input",
+      "inputType": "text",
+      "model": "ogDescription",
+      "label": "og:description",
+      "placeholder": "og:description",
+      "visible": "model.useDescription != true"
+    },
+    {
+      "type": "material-select",
+      "label": "og:type",
+      "model": "ogType",
+      "placeholder": "og:type",
+      "default": "website",
+      "values": [
+        {
+          "name": "website",
+          "value": "website"
+        }
+      ]
+    },
+    {
+      "type": "pathbrowser",
+      "browserRoot": "/content/assets",
+      "browserType": "image",
+      "label": "og:image",
+      "readonly": true,
```

Comment:
any reason why the og:image property would be read only? seems to not make any sense

Reply:
We used to do that to force the user into using the pathbrowser. But in the end we had to undo this because its nonsense. So we can remove that